main.cc

```diff
@@ -357,12 +357,39 @@ int main(int ac, char** av) {
     // If --version is requested, print it out and exit immediately to avoid
     // Seastar-specific warnings that may occur when running the app
     bpo::variables_map vm;
-    bpo::store(bpo::command_line_parser(ac, av).options(app.get_options_description()).allow_unregistered().run(), vm);
+    auto parsed_opts = bpo::command_line_parser(ac, av).options(app.get_options_description()).allow_unregistered().run();
+    bpo::store(parsed_opts, vm);
     if (vm["version"].as<bool>()) {
         fmt::print("{}\n", scylla_version());
         return 0;
     }
 
+    if (ac) {
+        fmt::print("Scylla executed using: \"{}", av[0]);
+        for (int i = 1; i < ac; ++i) {
+            fmt::print(" {}", av[i]);
+        }
+        fmt::print("\"\n");
+    }
+
+    fmt::print("Parsed command line options:\n");
```

Comment:
I wonder if this parsed view is helpful (does it say anything the user can't just see in the original command line you printed above?), but I guess it can't hurt.

Reply:
It's useful in a sense that it could help catch problems like the one in #4141 when some arguments are not parsed as expected.

Reply:
As @haaawk said. See also conversation with Avi below.